src/lib/util/cdrom.cpp

```diff
@@ -2283,270 +2367,266 @@ std::error_condition cdrom_file::parse_cue(std::string_view tocfname, toc &outto
+			{
+				/*
+				IsoBuster and ImgBurn (single bin file) - Lead-out time is the start of the lead-out
+				lead-out time - MSF of last track of session = size of last track
 
-					/* get lead-out time */
-					TOKENIZE
-					int leadout_offset = msf_to_frames(token);
-					outinfo.track[trknum].leadout = leadout_offset;
-				}
-				else if (!strncmp(linebuffer+i, "LEAD-IN", 7))
-				{
-					/*
-					IsoBuster and ImgBurn (single bin file) - Not used?
-					Redump and DiscImageCreator (multiple bins) - Lead-in time is the duration of just the lead-in
-					*/
-					TOKENIZE
+				Redump and DiscImageCreator (multiple bins) - Lead-out time is the duration of just the lead-out
+				*/
+				TOKENIZE
 
-					/* get lead-in time */
-					TOKENIZE
-					leadin = msf_to_frames(token);
-				}
-				else if (is_gdrom && !strncmp(linebuffer+i, "SINGLE-DENSITY AREA", 19))
-				{
-					/* single-density area starts LBA = 0 */
-					current_area = SINGLE_DENSITY;
-				}
-				else if (is_gdrom && !strncmp(linebuffer+i, "HIGH-DENSITY AREA", 17))
-				{
-					/* high-density area starts LBA = 45000 */
-					current_area = HIGH_DENSITY;
-				}
+				/* get lead-out time */
+				TOKENIZE
+				int leadout_offset = msf_to_frames(token);
+				outinfo.track[trknum].leadout = leadout_offset;
 			}
-			else if (!strcmp(token, "FILE"))
+			else if (!strncmp(linebuffer+i, "LEAD-IN", 7))
 			{
-				/* found the data file for a track */
+				/*
+				IsoBuster and ImgBurn (single bin file) - Not used?
+				Redump and DiscImageCreator (multiple bins) - Lead-in time is the duration of just the lead-in
+				*/
 				TOKENIZE
 
-				/* keep the filename */
-				std::string prevfname = lastfname;
-				lastfname.assign(path).append(token);
-				if (prevfname.length() > 0 && lastfname.compare(prevfname) != 0)
-					is_multibin = true;
-
-				/* get the file type */
+				/* get lead-in time */
 				TOKENIZE
+				leadin = msf_to_frames(token);
+			}
+			else if (is_gdrom && !strncmp(linebuffer+i, "SINGLE-DENSITY AREA", 19))
+			{
+				/* single-density area starts LBA = 0 */
+				current_area = SINGLE_DENSITY;
+			}
+			else if (is_gdrom && !strncmp(linebuffer+i, "HIGH-DENSITY AREA", 17))
+			{
+				/* high-density area starts LBA = 45000 */
+				current_area = HIGH_DENSITY;
+			}
+		}
+		else if (!strcmp(token, "FILE"))
+		{
+			/* found the data file for a track */
+			TOKENIZE
 
-				if (!strcmp(token, "BINARY"))
-				{
-					outinfo.track[trknum+1].swap = false;
-				}
-				else if (!strcmp(token, "MOTOROLA"))
-				{
-					outinfo.track[trknum+1].swap = true;
-				}
-				else if (!strcmp(token, "WAVE"))
-				{
-					wavlen = parse_wav_sample(lastfname, &wavoffs);
-					if (!wavlen)
-					{
-						fclose(infile);
-						printf("ERROR: couldn't read [%s] or not a valid .WAV\n", lastfname.c_str());
-						return chd_file::error::INVALID_DATA;
-					}
-				}
-				else
+			/* keep the filename */
+			std::string prevfname = lastfname;
+			lastfname.assign(path).append(token);
```

Comment:
Since you’re going to assign to `lastfname` immediately, you should move out of it – `std::string prevfname(std::move(lastfname));`